src/r_things.c

```diff
@@ -692,6 +692,10 @@ static void R_ProjectSprite (mobj_t* thing)
     }
   vis->brightmap = R_BrightmapForSprite(thing->sprite);
 
+  const byte *tmp_brightmap = R_BrightmapForState(thing->state - states);
+  if (tmp_brightmap != nobrightmap)
+    vis->brightmap = tmp_brightmap;
```

Comment:
Should we move the earlier `R_BrightmapForSprite()` call into the `else` branch, so we don't have to call two brightmaps functions during each call?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1750/commits/cd7147d8be6cd38fc9b9f4c606b687aaeba8dd3c